doc/spire_server.md

```diff
@@ -82,6 +82,7 @@ This may be useful for templating configuration files, for example across differ
 | `ratelimit`                  | Rate limiting configurations, usually used when the server is behind a load balancer (see below)                                                                                                                                                    |                                                                |
 | `socket_path`                | Path to bind the SPIRE Server API socket to (Unix only)                                                                                                                                                                                             | /tmp/spire-server/private/api.sock                             |
 | `trust_domain`               | The trust domain that this server belongs to (should be no more than 255 characters)                                                                                                                                                                |                                                                |
+| `use_legacy_downstream_x509_ca_ttl` | Use the downstream spire-server registration entry TTL as the downstream CA TTL. This will be deprecated in SPIRE 1.12.0                                                                                                                     | true                                                           |
```

Comment:
>This will be deprecated in SPIRE 1.12.0

Since this is already deprecated, I would change that to say:
This is deprecated and will be removed in a future version.